src/sangeeta/BankingSystem.java

```diff
@@ -0,0 +1,71 @@
+package sangeeta;
+
+public class BankingSystem {
+	String name;
+	int Totalbalance;
+	int DebitCount = 0;
+	int CreditCount = 0;
+	int PrintBalanceCount = 0;
+	static int TotalDebitCount = 0;
+	static int TotalCreditCount = 0;
+	static int TotalPrintBalanceCount = 0;
+	
+	void setData(int initialBalance, String cusName) {
+		Totalbalance = initialBalance;
+		name = cusName;
+	}
+	
+	void debitAmount(int amount) {
+		while(Totalbalance >= amount) {
```

Comment:
Why while loop here?
you should write if condition to check whether balance is sufficient or not.